selectionsort.py

```diff
@@ -0,0 +1,40 @@
+# Here we use dictionary data structure to store our count of values mapped to respective names in the above photos
+singers={"Selena Gomez": 112, "Taylor Swift": 178,"Ed Sheeran": 230,"Justin Bieber": 90,"John Legend": 190,"Beyoncé": 80}
+
+# We then convert dictionary to list of only count values and in singers_list, it will be like this[112,178,230,90,190,80]
+singers_list=list(singers.values())
+
+# Starting from here, we define a function for selection sort
+def selectionSort(singers_list):
+    #This is an empty list which will later be filled with sorted counting values
+    singers_rating = []
+    # This works to get length of singers' list,which is 6 and then loop until the end of index
+    for i in range(len(singers_list)): 
+        #We call findLargest function and search for largest value in the sigers' list and I recommend to look first at next third line
+        largest = findLargest(singers_list)
+        #We got the largest value by calling findLargest function below and we afterwards remove that largest value from singers_list and then add to our empty list declared above
+        singers_rating.append(singers_list.pop(largest))
```

Comment:
you can simply add singers_list[index] -> pop is not required here.

Reply:
sis, all this code exists in selection sort chapter of Grokking algorithm. What I changed is only names. Every line of code is the same 

Here is the source code
def findSmallest(arr):
 smallest = arr[0] Stores the smallest value
 smallest_index = 0 Stores the index of the smallest value
 for i in range(1, len(arr)):
 if arr[i] < smallest:
 smallest = arr[i]
 smallest_index = i
 return smallest_index
Now you can use this function to write selection sort:
def selectionSort(arr): Sorts an array
 newArr = []
 for i in range(len(arr)):
 smallest = findSmallest(arr)
 newArr.append(arr.pop(smallest))
 return newArr
print selectionSort([5, 3, 6, 2, 10])